src/index.ts

```diff
@@ -340,16 +279,7 @@ export default class Mnoga {
    * @param data  Contains data to be interpolated. `count` is a magic value for pluralization.
    * @returns     Best string match for key given.
    */
-  public t(key: string, data: TDataObject = {}, options: TOptions = {}): string {
-    // If we're not in production mode, be sure the people testing
-    if (process.env.NODE_ENV !== 'production') {
-      if (!key.match(VALID_KEY)) {
-        throw new Error(
-          `${key} is not a valid key format. ` +
-          `Valid keys are of the format ${VALID_KEY.toString()}`);
-      }
-    }
-
+  public t(key: string, data: TData = {}, options: TOptions = {}): string {
```

Comment:
Maybe there should be something in the code saying that `options` is intentionally undocumented, so someone doesn't see it and think we just forgot to document it?